Dockerfile

```diff
@@ -1,6 +1,6 @@
 ## Base image with node and entrypoint scripts ##
 ## =========================================== ##
-FROM node:10.16.3-alpine AS base
+FROM node:12.18.3-alpine AS base
```

Comment:
I recommend pinning to a specific version of alpine. If for some reason this tag is rebuilt with a different/newer alpine version, there's a chance it could break some dependencies (or maybe apk packages will be renamed/have different versions/etc). I've found that alpine package management is absolutely terrible, and pinning to a specific version of alpine helps _a little_